tools/homebrew-start/lanraragi

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+trap "kill 0" SIGINT
+
+# prefix/place where lrr is installed
+workdir=$(dirname $0)/$(dirname $(readlink $0))/..
+
+if [[ ${OSTYPE} == "darwin"* ]]; then
+  _LRR_DATA_DIRECTORY="${HOME}/Library/Application Support/LANraragi"
+else
+  _LRR_DATA_DIRECTORY="${HOME}/LANraragi"
+fi
+echo "Data Directory is set to '${_LRR_DATA_DIRECTORY}'"
+
+# modify redis config
+sed -i "" "s~<LRR_DATA_DIRECTORY>~${_LRR_DATA_DIRECTORY}~g" ${workdir}/redis.conf
+
+# create path if it does not exist yet
+mkdir -p ${_LRR_DATA_DIRECTORY}/log
```

Comment:
```suggestion
mkdir -p "${_LRR_DATA_DIRECTORY}/log"
```
The `Application Support` issue pops up here too, the log folder wasn't being created on my mac test machine.

Reply:
(I think this really is the last issue and hopefully I can go sleep after that)

Reply:
ah yeah, forgot the quotes here too